api/hydra/api.ttl

```diff
@@ -22,7 +22,7 @@
     dataCube:Column ,
     api:Error ,
     api:ProjectPlaceholder ,
-    dataCube:ColumnAttribute ,
+    dataCube:ValueAttribute ,
```

Comment:
Potential breaking change: I renamed the attribute as we called it "value attribute" wit hAdrian